app.go

```diff
@@ -123,9 +153,14 @@ func (a *APIApp) Run(ctx context.Context) error {
 		grip.Debug(srv.Shutdown(ctx))
 	}()
 
-	<-serviceWait
+	wait := func(wctx context.Context) {
+		select {
+		case <-wctx.Done():
```

Comment:
Does this stop `ListenAndServe()`?

Reply:
no, it doesn't. you have to cancel the context you pass to the function.

Reply:
this just cancels waiting.